docs/applications/cloud-storage/install-nextcloud-13.md

```diff
@@ -0,0 +1,122 @@
+---
+author:
+  name: Angel
+  email: docs@linode.com
+description: "Nextcloud is an open source solution to hosting your own content online. In addition to the total control users gain over their own files, Nextcloud offers customizable security features that allow the user to take control of sharing and access privileges."
+og_description: "Nextcloud 13 Brings Improved UI, video and text chat, and end-to-end encryption wrapped into a cloud storage platform. This guide will walk you through installing Nextcloud 13 using Docker."
+keywords: ["nextcloud", "cloud", "open source hosting", "video chat"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2017-03-14
+modified: 2018-03-07
+modified_by:
+  name: Angel
+title: 'Install NextCloud 13'
+external_resources:
+  - '[Nextcloud Docker Image Documentation](https://github.com/nextcloud/docker)'
+---
+
+Nextcloud 13 is a cloud storage platform that also offers users the ability to self-host a video and text chat platform, featuring end-to-end encryption. This guide will walk you through setting up Nextcloud, and show how to use the video chat platform built into the latest release.
+
+## Install Docker CE
+
+You will need a Linode with Docker CE installed to follow along with the steps in this guide.
+
+{{< content "install_docker_ce.md" >}}
+
+## Install Nextcloud 13 and Talk
+
+### Nextcloud
+
+1. Pull and run the Nextcloud image:
+
+        docker run -d -p 8080:80 nextcloud
+
+2.  In a browser, navigate to port 8080 on your Linode (e.g. `192.0.2.0:8080`) to launch the Nextcloud console.
+
+3. Create an admin account when prompted:
+
+![Admin_account_creation](/docs/assets/docker_nextcloud/admin_creation.png)
+
+### Talk
+
+**Talk** is the video chat platform add-on offered by Nextcloud. It's built on [Spreed](https://github.com/nextcloud/spreed).
+
+1. From the Nextcloud console main page, click the **Settings** icon on the right side of the navigation bar. Choose **+ Apps**.
+
+2.  Install the Talk add-on. It is located in the **Social & communication** section; select the app and click **Enable**.
+
+![talk_addon](/docs/assets/docker_nextcloud/talk_addon.png)
+
+### How to Use Talk
+
+Nextcloud 13 offers a full suite of addons. Talk works by allowing all the users that are registered to your Nextcloud instance to communicate with each other. Nextcloud Talk offers simple text chat, private or group password protected calls, and screen sharing. Nextcloud Talk is built using [WebRTC](https://simplewebrtc.com/), and works in your browser. Navigate to the **Users** section of the Nextcloud interface, and create logins for your team to use. After the logins are distributed to your team members, they will be able to communicate with each other.
```

Comment:
Older Nextcloud installation guides need to have versions explicitly stated since video chat is available for version 13+